tests/plugins/widgetselection/keys.js

```diff
@@ -4,12 +4,36 @@
 ( function() {
 	'use strict';
 
-	bender.editor = true;
+	bender.editors = {
+		editor1: {
```

Comment:
The name doesn't tell much. Maybe `classic`?

Reply:
Addressed in 84265bd798.